pom.xml

```diff
@@ -395,9 +395,6 @@
                   <file>${project.build.directory}/${project.artifactId}-${javacpp.platform}${javacpp.platform.extension}.jar</file>
                   <moduleInfoSource>
                     open module org.bytedeco.${javacpp.packageName}.${javacpp.platform.module} {
-                      requires transitive org.bytedeco.${javacpp.packageName};
```

Comment:
I think we should leave the first one here though. And we should also remove `<javacpp.nativeRequires/>`